packages/mdc-layout-grid/mdc-layout-grid.scss

```diff
@@ -79,6 +79,17 @@
       @include mdc-layout-grid-cell-order($i);
     }
   }
+    
+  //Order per device.
+  @each $size in map-keys($mdc-layout-grid-breakpoints) {
```

Comment:
Can you please provide some context about this change?

Thanks!

Reply:
This extends the current `mdc-layout-grid__cell--order-<INDEX>` with `-<TYPE_OF_DEVICE>`.
It makes it possible to have a different order of elements on different types of devices.

In my current project I'm using this when on desktop I want to have two cells with span 6. On phone I want to display both cells with a span of 4, but the second cell first.

Using  the current `mdc-layout-grid__cell--order-<INDEX>` would mean the second cell would also show first (left) on desktop, or last on phone (The same as no order). Wich is not what I want.

Thanks!

Reply:
+1 for this, having an order class doesn't make much sense when you can't even use it to change the layout based on available size